tests/integration_tests/test_sampler.py

```diff
@@ -0,0 +1,160 @@
+from mock import patch
+import pytest
+
+import optuna
+from optuna.integration import CmaEsSampler
+from optuna.integration import SkoptSampler
+from optuna.samplers import BaseSampler  # NOQA
```

Comment:
IMO. The types that used only for type checking should be imported only when `optuna.types.TYPE_CHECKING` is `True`.

Reply:
Sorry for my careless mistake. I fixed it in https://github.com/pfnet/optuna/pull/447/commits/dfce2812e3130bcab5611c094e527566d6dcfeeb.